### ISSUE TITLE
Max body size units

### ISSUE THREAD

**Issue by mikicz** [jazzband/django-silk#300]
I have a question regarding the `SILKY_MAX_REQUEST_BODY_SIZE` and `SILKY_MAX_RESPONSE_BODY_SIZE` settings.

The README example is following:

```
SILKY_MAX_REQUEST_BODY_SIZE = -1  # Silk takes anything <0 as no limit
SILKY_MAX_RESPONSE_BODY_SIZE = 1024  # If response body>1024kb, ignore
```

From that comment I inferred that the unit of the setting should be kB, However then I took a look at the source code and `sys.getsizeof` is used to get the size of the bodies. Doesn't that function return number of bytes and not number of kilobytes? 

In what unit should the setting be set? Kilobytes or bytes? In anycase I'd be happy to send a PR making this clearer in the README or updating the comparison in the ModelFactories to take the setting as kB...

**Comment by SebCorbin** [comment]
Bytes, this is a documentation issue

**Comment by albertyw** [comment]
Fixed in d40bb9dc
